docs/APIRef.DeviceObjectAPI.md

```diff
@@ -204,8 +204,10 @@ Currently only available in the iOS Simulator.<br>
 Check out Detox's [own test suite](../detox/test/e2e/06.device-orientation.test.js)
 
 ### `device.setLocation(lat, lon)`
->Note: `setLocation` is dependent on `fbsimctl`. if `fbsimctl` is not installed, the command will fail, asking for it to be installed.
-Sets the simulator location to the given latitude and longitude.
+>Note:
+On iOS `setLocation` is dependent on `fbsimctl`. if `fbsimctl` is not installed, the command will fail, asking for it to be installed.
+On Android `setLocation` is dependent on your tests running on the Android Emulator (with the correct permissions set in your manifest), **not** on a physical device.
```

Comment:
I'm not sure the phrasing is clear here:
> is **dependent on your tests** running on the Android Emulator.

Of course, it is. What did you mean? If I got the message right, you could just rephrase that `setLocation` currently is supported only for Android emulator. And also I think that this notice is applicable not only to a physical device but, for example, to Genymotion virtual devices as well (they'll require another implementation). 

Also, did you mean here:
> with the correct permissions set in your manifest

the app manifest? If so, good to clarify that.